corehq/apps/reports/filters/controllers.py

```diff
@@ -125,7 +126,8 @@ def _get_users(self, query, start, size, include_inactive=False):
     def active_user_es_query(self, query):
         search_fields = ["first_name", "last_name", "base_username"]
         return (UserES()
-                .domain(self.domain)
+                .domain(self.domain, allow_enterprise_controlled_domains=EnterprisePermissions.
```

Comment:
Couldn't `allow_enterprise_controlled_domains` just be True, since the `domain` function will check the value of `is_enabled`?

Reply:
ah yes 👍